charts/kyverno-policies/templates/pod-sec-ctxt-cray-bos.yaml

```diff
@@ -24,7 +24,6 @@ spec:
         selector:
           matchLabels:
             app.kubernetes.io/instance: cray-bos
-            app.kubernetes.io/name: cray-bos
```

Comment:
A matching validation policy exists that needs to mirror this change. See https://github.com/Cray-HPE/cray-kyverno-policies/blob/CASMPET-5501/charts/kyverno-policies/templates/validation/validate-pod-sec-ctxt-cray-bos.yaml.

Reply:
I modified the validation policy, too. Thanks.